<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GyroScope</class>
 <widget class="QWidget" name="GyroScope">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1431</width>
    <height>979</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777213</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>40</y>
     <width>281</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>841</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCustomPlot" name="Acc_X_customPlot" native="true"/>
      </item>
      <item>
       <widget class="QCustomPlot" name="Acc_Y_customPlot" native="true"/>
      </item>
      <item>
       <widget class="QCustomPlot" name="Acc_Z_customPlot" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCustomPlot" name="Gyro_X_customPlot" native="true"/>
      </item>
      <item>
       <widget class="QCustomPlot" name="Gyro_Y_customPlot" native="true"/>
      </item>
      <item>
       <widget class="QCustomPlot" name="Gyro_Z_customPlot" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustom./qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
